### ISSUE TITLE
Item flag "vanilla" not work.

### ISSUE THREAD

**Issue by momoservertw** [RockinChaos/ItemJoin#361]
Click the item with flag "vanilla" will still run commands and other item flags seem to be still available.

**Video**
https://youtu.be/2q5QQ9PtiKg

**Pastebin**
https://pastebin.com/VRxF8SVd

**Version**
Paper version: git-Paper-439 (MC: 1.16.5) (Implementing API version 1.16.5-R0.1-SNAPSHOT)
ItemJoin version: v5.1.1-SNAPSHOT-b664

**Additional context**
Using ItemJoin is very useful to create custom item. The configuration is easy to setup. haha
According the wiki depiction, the flag "unbreakable" will may be invalid, but I want use it.
Can you also check it? XD
```
vanilla - Allows the giving of default unmodified no-name items. Please note if you choose to use commands or itemflags, etc. They will not function since all this does is allow ItemJoin to give a vanilla type item that can have a modified name or lore if defined.
```

**Comment by RockinChaos** [comment]
Lol, I just love when a singular space that didn't use to be there just magically appears.
![image](https://user-images.githubusercontent.com/10971577/106987143-a6813780-6721-11eb-8f19-7eaed5b84581.png)

Removing this space fixed the issue :) haha.

As for the itemflags, I will need to modify the wiki but only SOME itemflags will be made invalid, ones that are MODIFIERS will still apply. Such as attributes/unbreakable/always-give. Mainly the ones that stop working are self-drops, death-drops, inventory-modify etc. (Event itemflags). Hope that makes sense, ill probably need to write that in a more coherent way in the wiki.

Link to fixed snapshot; https://ci.craftationgaming.com/job/ItemJoin/667/
**Remember to remove your old ItemJoin.jar file**

**Comment by RockinChaos** [comment]
This bug has been officially fixed in the release of ItemJoin v5.2.0.
You can grab it here; https://www.spigotmc.org/resources/itemjoin.12661/download?version=383801

Please read the changelog as the documentation has not been updated yet;
https://github.com/RockinChaos/ItemJoin/wiki/Recent-Changes
